packages/react/src/constants/CardPropTypes.js

```diff
@@ -966,6 +966,7 @@ export const CardPropTypes = {
         hidden: PropTypes.bool,
       })
     ),
+    content: PropTypes.arrayOf(PropTypes.node),
```

Comment:
This should probably reflect the option of a single object which is the current prop signature?

Reply:
Yeah I changed it to be just the node... and also some other small updates